air-quality-ui/system_tests/utils/helper_methods.ts

```diff
@@ -73,3 +77,38 @@ export async function waitForIdleNetwork(
     throw error
   }
 }
+
+export async function verifyForecastAndMeasurements(
```

Comment:
I don't think this needs to be split out into a function in here. Instead in your test file, I would make two tests, one for forecast and one for measurement requests. I'd then have your assertion based on the whole API call rather than sections of it (see `summary_page_api_calls.spec.ts`). This way you keep the test file for assertions